.github/workflows/ci.yaml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
pull_request alone should suffice here.

```
on:
  pull_request:
    types: [opened, reopened, synchronize]
```

Reply:
we don't need to run both push and pull request here. hopefully this will fix it
<img width="899" alt="image" src="https://github.com/vatz88/FFCSonTheGo/assets/16202323/d7b93374-b172-4d6d-a9f5-f2e696cc1fde">